atomsci/ddm/pipeline/model_wrapper.py

```diff
@@ -335,7 +335,7 @@ def _create_output_transformers(self, model_dataset):
                 transformers: A list of deepchem transformation objects on response_col, only if conditions are met
         """
         # TODO: Just a warning, we may have response transformers for classification datasets in the future
-        if self.params.prediction_type=='regression' and self.params.transformers==True:
+        if self.params.prediction_type=='regression' and self.params.transformers:
```

Comment:
```suggestion
        if self.params.prediction_type=='regression' and self.params.transformers is True:
```

I was talking to @SaiedaJN about this, and while I _think_ this is OK, if we want to preserve the behavior of the code 100%, we should keep these as identify checks. At a later time the AMPL team can remove these for simpler truthy/falsey checks